libs/blocks/global-navigation/utilities/menu/menu.js

```diff
@@ -133,12 +133,9 @@ const decoratePromo = (elem, index) => {
   const imageElem = elem.querySelector('picture');
 
   if (!isImageOnly) {
```

Comment:
One improvement idea: do we still need the `isImageOnly` check, or could otherwise apply different paddings based on the class in CSS?

Reply:
I thought about this too, but decided to keep it, as that modifier affects logic, not styling. As you can see in the last 2 dropdown menus from the Milo test link, the modifier hides content and applies whatever link the content contains to the image itself. In this case, since there won't be any content, there's no use to execute this logic. Styling is not affected overall, both situations work